### ISSUE TITLE
FR: Add a `FinalScripts` section which will run after all other items

### ISSUE THREAD

**Issue by BigMacAdmin** [SecondSonConsulting/Baseline#65]
Intended to be used for Webhooks to report Baseline status, or other "absolulte final cleanup" type tasks.

These will run immediately before the "DialogFailWindow" or "DialogSuccessWindow" time period, so the documentation should imply that they need to be very quick to run.

**Comment by keepfacesbeard** [comment]
Are you thinking this will leverage the process_scripts function, and reference a new key in the config profile? Running right after: `dialog_command "quit:" `

**Comment by BigMacAdmin** [comment]
Yes, the same `process_scripts` function would be used, no need to rewrite any of it I don't think.

I think it needs to fire off prior to `dialog_command quit:` that way it doesn't cause a delay between the primary window and the success/fail window.

If you're considering a PR, [please review this page](https://github.com/SecondSonConsulting/Baseline/wiki/Contributing-to-Baseline) :) thanks.

**Comment by BigMacAdmin** [comment]
@keepfacesbeard What do you think about having two separate sections, one that runs if all items were successful and one that runs if there were failures?
 
 To help people keep chatter down in case they only want to know about failed items, etc.
 
 Thoughts?

**Comment by keepfacesbeard** [comment]
So two separate function calls, like  if[success]; FinalScriptsSuccess, elif[failure]; FinalScriptsFailure, dependent on whether or not there's a failure of any kind?
Failures in scripts/packages is basically the kind of reporting I am trying to do in my FinalScript webhook script, but I figured any logic like that could be built into the script itself. Some folks might want specific things to get called out, script failures, logs, etc., or send to different places depending on failure mode.  I'm not sure about the added fail/success separation. No strong opinion, I suppose. Some people might find it useful.

**Comment by BigMacAdmin** [comment]
Added in 2.3beta1
